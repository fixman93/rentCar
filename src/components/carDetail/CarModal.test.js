// import React from 'react'
// import Adapter from 'enzyme-adapter-react-16';

// import { shallow, configure } from 'enzyme'

// import { CarModal } from './CarModal'

// configure({ adapter: new Adapter() });

// describe('CarModal', () => {

//   const carmodal = shallow(<CarModal />)
//   it('render carModal', () => {
//     expect(carmodal).toMatchSnapshot()
//   })





// })