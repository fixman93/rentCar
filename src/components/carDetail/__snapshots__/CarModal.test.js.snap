// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CarModal render carModal 1`] = `ShallowWrapper {}`;
